Controllers/CartController.cs

```diff
@@ -117,6 +118,41 @@ public CartController(BangazonContext ctx)
                      new { controller = "Products", action = "Detail", Id = id } ) );
         }
 
+        public async Task<IActionResult> DeleteLineItem([FromRoute]int id)
+        {
+            if (SessionHelper.ActiveUser == null)
```

Comment:
This check isn't needed. User won't be able to view a cart at all if they aren't logged in